app/client/src/components/pages/StateTribal.Tabs.WaterQualityOverview.SiteSpecific.js

```diff
@@ -441,6 +442,13 @@ function SiteSpecific({
             }
           >
             <div css={accordionContentStyles}>
+              <DisclaimerModal>
+                <p>
+                  The impairments listed below do not necessarily affect the{' '}
+                  <strong>{useSelected}</strong> use but are present in the
```

Comment:
It doesn't look like it's done sitewide, but "use" is capitalized in the ticket since it's an ATTAINS "keyword".
```suggestion
                  <strong>{useSelected}</strong> Use but are present in the
```

Reply:
I'll fix that.